hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenRejectHandler.java

```diff
@@ -62,7 +70,51 @@ public void handle(@NonNull final HandleContext context) throws HandleException
     @NonNull
     @Override
     public Fees calculateFees(final FeeContext feeContext) {
-        // Todo: Implement the logic for calculating fees
-        return Fees.FREE;
+        final var body = feeContext.body();
+        final var op = body.tokenRejectOrThrow();
```

Comment:
We could probably have. abase class for `CryptoTransferHandler` and `TokenRejectHandler` and re-use all code that is common.